onnxruntime/python/tools/transformers/models/whisper/whisper_chain.py

```diff
@@ -1,12 +1,19 @@
 import logging
 import os
+import sys
 
 import onnx
-from benchmark_helper import Precision
-from convert_generation import get_shared_initializers, update_decoder_subgraph_share_buffer_and_use_decoder_masked_mha
 from onnx import TensorProto, helper
 from transformers import WhisperConfig
 
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", ".."))
```

Comment:
[This PR](https://github.com/microsoft/onnxruntime/pull/17179) removed the path appends. We can remove this and call the Whisper scripts from the `transformers` folder (e.g. `python -m models.whisper.convert_to_onnx ...`).

Reply:
Cool!